### PR TITLE
FileImage

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: packages/flutter/lib/src/widgets/image.dart [hunk 1]
```diff
@@ -90,6 +91,23 @@ class Image extends StatefulWidget {
   }) : image = new NetworkImage(src, scale: scale),
        super(key: key);
 
+  /// Creates a widget that displays an [ImageStream] obtained from a [File].
+  ///
+  /// The [file], [scale], and [repeat] arguments must not be null.
+  Image.file(File file, {
```
Comment (sethladd) [inline packages/flutter/lib/src/widgets/image.dart]: Can we add a note up in the class-level docs as well? The class-level docs mention the other constructors by name.

Comment (Hixie) [inline packages/flutter/lib/src/widgets/image.dart]: added

**Comment by sethladd** [comment]
Thanks!

**Comment by sethladd** [comment]
closes https://github.com/flutter/flutter/issues/6852

**Comment by abarth** [comment]
LGTM

**Comment by FaisalAbid** [comment]
lovee ittttt

**Pull request merged**